src/game.cpp

```diff
@@ -13116,53 +13161,116 @@ void game::climb_down( const tripoint &examp )
+            }
+        } else if( has_grapnel ) {
+            std::string query = _( "Use your grappling hook to climb down?" );
+            query += "%s";
+            if( !query_yn( query.c_str(), hint_remaining_fall ) ) {
                 has_grapnel = false;
             } else {
                 web_rappel = false;
             }
         }
 
-        if( !has_grapnel ) {
-            const char *query;
+        if( !can_use_ladder && !has_grapnel ) {
+            std::string query;
+            std::string hint_slip_risk;
+            std::string hint_fall_damage;
+            std::string hint_climb_back;
             if( web_rappel ) {
-                query = _( "Use your webs to descend?" );
+                query = string_format( n_gettext( "Use your webs to descend?",
+                                                  "Looks like %d stories.  Nothing your webs can't handle.  Descend?", height ), height );
             } else {
-                if( climb_cost <= 0 && fall_mod > 0.8 ) {
-                    query = _( "You probably won't be able to get up and jumping down may hurt.  Jump?" );
-                } else if( climb_cost <= 0 ) {
-                    query = _( "You probably won't be able to get back up.  Climb down?" );
+                // The most common query reads as follows:
+                //   It [seems somewhat risky] to climb down like this.
+                //   Falling [would hurt].
+                //   You [probably won't be able to climb back up].
+                query = "%s%s\n%s\n%s\n";
+                query += _( "Climb down?" );
+
+                // Calculate chance of slipping.  Prints possible causes to log.
+                int slip_chance = slip_down_chance( true );
+
+                // Roughly estimate damage if we should fall.
+                int damage_estimate = 10 * height;
+                if( damage_estimate <= 30 ) {
+                    damage_estimate *= fall_mod;
+                } else {
+                    damage_estimate *= std::pow( fall_mod, 30.f / damage_estimate );
+                }
+
+                // Rough messaging about safety.  "very small risk" may or may not be zero chance.
+                if( slip_chance < 3 ) {
+                    hint_slip_risk = _( "It <color_green>seems safe</color> to climb down like this." );
+                } else if( slip_chance < 8 ) {
+                    hint_slip_risk = _( "It <color_yellow>seems a bit tricky</color> to climb down like this." );
+                } else if( slip_chance < 20 ) {
+                    hint_slip_risk = _( "It <color_yellow>seems somewhat risky</color> to climb down like this." );
+                } else if( slip_chance < 50 ) {
+                    hint_slip_risk = _( "It <color_red>seems very risky</color> to climb down like this." );
+                } else if( slip_chance < 80 ) {
+                    hint_slip_risk = _( "It <color_pink>looks like you'll slip</color> if you climb down like this." );
```

Comment:
The "like this" phrasing only makes sense contextually if you see the prompts for using a ladder / webs / grapple and refuse them. If you have no ladder, webs, or grapple, then it sounds kind of out of place imo.

Reply:
I chose "like this" rather than "here" to draw attention to the player's encumbrance, stamina, wetness, free hands and other slip chance factors.  These are printed to the message log before the prompt, giving the player a chance to realize their mistake before attempting e.g. a low-stamina climb.

Reply:
TBH not sure how much someone will notice the extra info in the sidebar even if they have the message log visible by default, but okay.

Reply:
Yeah.  It would be interesting if there were some way to put the log messages from `slip_down_chance` into the prompt in the future.